quill-core/src/main/scala/io/getquill/context/ContextMacro.scala

```diff
@@ -65,7 +65,7 @@ trait ContextMacro extends Quotation {
 
         ProbeStatement(idiom.prepareForProbing(string), c)
 
-        c.info(string)
```

Comment:
> this is just a compilation info message, it doesn't produce logging at run time

Oh, I see. I made a mistake.

As loggers level in the other context, can I change it to debug?